consensus/core/src/config/mod.rs

```diff
@@ -26,11 +26,19 @@ pub struct Config {
     // TODO:
     // is_archival: bool,
     // enable_sanity_check_pruning_utxoset: bool,
+    //
+    /// Enable the UTXO index
+    pub utxoindex: bool,
```

Comment:
This is consensus config. It's a little bit weird to put it here. Maybe add a more general config struct that wraps it?

Reply:
yes, let's create a dedicated internal crate for that

Reply:
perhaps keep as is now and add a TODO

Reply:
good for me